### DISCUSSION TITLE
How to install php extention?

### DISCUSSION THREAD

**Discussion by yrsdi** [asdf-community/asdf-php#74] [Q&A] [answered]
i need to install php-curl and other extention. How to install it in different version of php?

**Comment by smorimoto** [comment, marked as answer]
Please search for existing issues and discussions before asking questions. Didn't these help you? https://github.com/asdf-community/asdf-php/discussions/75 https://github.com/asdf-community/asdf-php/discussions/76

**Reply by yrsdi** [reply]
ok thanks. it's work

**Reply by smorimoto** [reply]
No worries, great.
